samples/svg/src/main/res/drawable/image_loading.xml

```diff
@@ -1,12 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <layer-list xmlns:android="http://schemas.android.com/apk/res/android">
-    <item>
-        <bitmap
-            android:src="@android:drawable/sym_def_app_icon"
-            android:gravity="center" />
```

Comment:
According to the official docs this resource is available in O. It would be a breaking change for them to remove it without deprecation first. Maybe the preview is missing it (in which case report an AOSP bug), I would wait until a final release before taking action.